frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx

```diff
@@ -49,6 +50,7 @@ export const taxonomicFilterLogic = kea<taxonomicFilterLogicType>({
             groupPropertiesModel,
             ['allGroupProperties'],
         ],
+        actions: [columnConfiguratorLogic, ['save']],
```

Comment:
Oohh... no, this won't do. It's like a textfield has a dependency on a settings page. It should be the other way around.

Reply:
Yeah, I've been sat chewing this over. There are some frustrating consequences of taxonomic filter being inside the column configurator that I don't seem to be able to squash